docs/understand/enterprise/index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: lakeFS Enterprise
+description: lakeFS Enterprise is an enterprise-ready lakeFS solution providing additional features including RBAC, SSO and Support SLA.
+has_children: true
+has_toc: false
+parent: Understanding lakeFS
+redirect_from:
+- /understand/lakefs-enterprise.html
+- /enterprise/index.html
+---
+
+# lakeFS Enterprise
+
+lakeFS Enterprise is an enterprise-ready lakeFS solution that provides a support SLA and additional features to the open-source version of lakeFS. The additional features are:
+
+* [RBAC]({% link reference/security/rbac.md %})
+* [SSO]({% link reference/security/sso.md %})
+* [STS Auth]({% link reference/security/sts-login.md %})
+* [External Principals AWS Auth]({% link reference/security/external-principals-aws.md %})
```

Comment:
I really like this approach!

I would try to break it a bit more. For example:

```
LakeFS Enterprise is a commercially-supported version of the open-source lakeFS project, offering additional features and functionalities targeted for businesses.  It provides several benefits over the open-source version:

* Security - Advanced Authorization 
  * [RBAC]({% link reference/security/rbac.md %}) -  implements role-based access control to manage user permissions. It allows for fine-grained control by associating permissions with users and groups, granting them specific actions on specific resources. This ensures data security and compliance within an organization.
* Security - Advanced Authentication
  * [SSO]({% link reference/security/sso.md %}) -  lets users sign in with existing credentials from a trusted provider, eliminating separate logins.
  * [STS Auth]({% link reference/security/sts-login.md %}) - offers temporary, secure logins using an Identity Provider, simplifying user access and enhancing security.
  * * [Authenticate to lakeFS with AWS IAM Roles]({% link reference/security/external-principals-aws.md %}) - lets programs authenticate using AWS IAM roles instead of lakeFS credentials, granting access based on IAM policies.
* Support SLA
```

Reply:
May be add: Contact Sales (https://lakefs.io/contact-sales/) to get the token for Fluffy.

Reply:
Why does last line say: "follow the examples in the [quickstart guide]" while the link points to "understand/enterprise/orchestration.md"